app/src/main/java/za/org/grassroot2/view/activity/CreateVoteActivity.kt

```diff
@@ -0,0 +1,52 @@
+package za.org.grassroot2.view.activity
```

Comment:
Do you need separate activities? Can't you do one activity with a type parameter?

Reply:
Okay this needs a rewrite. To be done, collaboratively.